internal/cmd/controller/reconciler/bundle_controller.go

```diff
@@ -664,6 +668,10 @@ func (r *BundleReconciler) maybeDeleteOCIArtifact(ctx context.Context, bundle *f
 		return err
 	}
 	err = ocistorage.NewOCIWrapper().DeleteManifest(ctx, opts, bundle.Spec.ContentsID)
+	if err != nil {
+		r.Recorder.Event(bundle, fleetevent.Warning, "Failed", fmt.Sprintf("deleting OCI artifact: %q, error: %v", bundle.Spec.ContentsID, err.Error()))
```

Comment:
Same remark/question as in `apply.go` on the event reason.

Reply:
changed to the same suggestion